src/System.Private.CoreLib/shared/System/SpanHelpers.cs

```diff
@@ -23,7 +23,7 @@ public static unsafe void ClearWithoutReferences(ref byte b, nuint byteLength)
             if (byteLength == 0)
                 return;
 
-#if CORECLR && (AMD64 || ARM64)
+#if AMD64 || ARM64
```

Comment:
Could you make this `!PROJECTN && (AMD64 || ARM64)`?

Reply:
sure, done